traffic_ops/traffic_ops_golang/cdn/configs.go

```diff
@@ -20,37 +20,33 @@ package cdn
  */
 
 import (
-	"database/sql"
-	"errors"
-	"net/http"
-
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
 )
 
-func GetConfigs(w http.ResponseWriter, r *http.Request) {
-	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
-	if userErr != nil || sysErr != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
-		return
-	}
-	defer inf.Close()
-	api.RespWriter(w, r, inf.Tx.Tx)(getConfigs(inf.Tx.Tx))
+//we need a type alias to define functions on
```

Comment:
nit: GoDocs should be complete sentences starting with the name of the object they describe.